test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Order.html

```diff
@@ -41,6 +41,9 @@
                 <option value="Credit">Credit</option>
                 <option value="Check" selected="selected">Check</option>
             </select>
+            <select>
+                <option value="Check">Check</option>
+            </select>
```

Comment:
No, there's no good way to verify that the context doesn't flow from one to the other. Reason being a lot of what we're verifying is due to the overall `TagHelper` execution flow when going across two separate `TagHelper`s.